infrastructure-playbooks/cephadm-adopt.yml

```diff
@@ -389,7 +389,7 @@
         CEPHADM_IMAGE: '{{ ceph_docker_registry }}/{{ ceph_docker_image }}:{{ ceph_docker_image_tag }}'
 
 - name: adopt ceph mgr daemons
-  hosts: "{{ mgr_group_name|default('mgrs') }}"
+  hosts: "{{ groups[mgr_group_name] | default(groups[mon_group_name]) }}"
```

Comment:
I'm not sure if this is working, if you remember using variables for group name in the `hosts` statement wasn't working at all and that's why we need a static default value.

I think a better solution would be to copy the mgr adoption tasks inside the mon adoption block like we're doing in the main playbook [1]

What do you think ?

[1] https://github.com/ceph/ceph-ansible/blob/master/site.yml.sample#L106-L108

Reply:
> I'm not sure if this is working, if you remember using variables for group name in the `hosts` statement wasn't working at all and that's why we need a static default value.

it works maggle